### PR TITLE
Update instructions in the README

### PR DESCRIPTION
**Pull request by PatrykMis**
Update build instructions for command line, fix typo.

### CODE REVIEW

File: README.md
Comment (Irineu333) [inline README.md:1]: Verry good

**Review by Irineu333 — APPROVED** [review]

**Comment by Irineu333** [comment]
I made some modifications directly in develop, I removed the instructions on how to run it on the command line. The instructions on how to do this in the IDE are enough to comply with the GPL license clause.

**Comment by Irineu333** [comment]
Sorry, I didn't know you mastered this.

**Comment by Irineu333** [comment]
Question: is it easier for someone who can't see to develop in an IDE or on the command line?

**Comment by PatrykMis** [comment]
Of course - Command line and editors like Notepad++, VisualCodium etc.

**Comment by Irineu333** [comment]
I understand, the instructions for building on the command line really cannot be missing.

**Comment by Irineu333** [comment]
I came back because seeing those instructions, I thought it couldn't be that simple. There seemed to be something missing.

**Comment by PatrykMis** [comment]
ANDROID_HOME should be defined and cmdline-tools should be installed properly (for sdkmanager script). If do, the Gradle downloads everything that's needed for the build automatically due to Android Gradle Plugin nature, really.

**Comment by akash07k** [comment]
Yep, most of the times, Commandline way is better for visually impaired devs as opposed to their GUI/IDE counterparts.
If we talk about the IntelliJ/AndroidStudio as @PatrykMis  said, they are far from perfect and commandline is the only way where we can build and debug effectively and efficiently.
> I understand, so the instructions for building on the command line really cannot be missing.



> Of course - Command line and editors like [Notepad++](https://github.com/notepad-plus-plus/notepad-plus-plus), [VSCodium](https://github.com/VSCodium/vscodium) etc.
> 
> Unfortunately, IDEs like InteliJ Idea and/or it's fork, Android Studio, is far from perfect regarding accessibility. The situation is changing from version to version, but they're not as good as, let's say, VSCodium even in 50%.
> 
> More background can give you visually impaired but experienced Android and DotNet developer @akash07k

**Comment by Irineu333** [comment]
@akash07k and @PatrykMis, Thank you very much for the explanation.

**Comment by PatrykMis** [comment]
> I removed it because there seemed to be something missing, and also the instruction did not install the app, but assemble it. The instructions must go step by step until installing the app.

To install the apk, the equivalent command is `gradlew installDebug` instead of `gradlew build`.

**Comment by Irineu333** [comment]
I opened a pull request to include it again.

**Pull request merged**